scripts/agent.yml

```diff
@@ -192,12 +195,25 @@ tasks:
           aws_key: ${aws_key}
           aws_secret: ${aws_secret}
           s3_copy_files:
-              - {'source': {'path': 'evergreen/smoke/${build_id}-${build_variant}/evergreen-${task_name}-${revision}', 'bucket': 'mciuploads'},
-                'destination': {'path': 'evergreen/smoke/${build_id}-${build_variant}/evergreen-${task_name}-${revision}-copy', 'bucket': 'mciuploads'}}
+            - {
+                "source":
+                  {
```

Comment:
If we're going to unroll these lines, can we just use YAML syntax here instead of JSON?

Reply:
Why would that be helpful?
I don't particularly want to unroll it, the editor does it automatically.

Reply:
that makes a lot of sense, thanks! 
done

Reply:
Looks good, I think there's just a couple YAML syntax errors.

Reply:
thanks! fixed